README.md

```diff
@@ -2,17 +2,28 @@
   <a href="https://www.pulumi.com?utm_campaign=pulumi-esc-github-repo&utm_source=github.com&utm_medium=top-logo" title="Pulumi ESC: Open source secrets management solution">
     <img src="https://www.pulumi.com/images/logo/logo-on-white-box.svg?" width="350">
    </a>
-
-  [![License](https://img.shields.io/github/license/pulumi/pulumi)](LICENSE)
-  [![Slack](http://www.pulumi.com/images/docs/badges/slack.svg)](https://slack.pulumi.com?utm_campaign=pulumi-esc-github-repo&utm_source=github.com&utm_medium=slack-badge)
-  [![X (formerly Twitter) Follow](https://img.shields.io/twitter/follow/PulumiCorp)](https://x.com/PulumiCorp)
 </p>
 
-# Pulumi ESC (Environments, Secrets, and Configuration)
+# Secrets Management for Multi-Cloud Environments
+
+**[Pulumi ESC](https://www.pulumi.com/product/esc/?utm_source=GitHub.com&utm_medium=README&utm_campaign=Pulumi+ESC+GitHub+Repo&utm_content=Intro)** is a centralized secrets management & orchestration service that makes it easy to tame secrets sprawl and configuration complexity securely across all your cloud infrastructure and applications. You can pull and sync secrets with any secrets store – including HashiCorp Vault, AWS Secrets Manager, Azure Key Vault, GCP Secret Manager, 1Password, and more – and consume secrets in any application, tool, or CI/CD platform.
+
+Pulumi ESC makes adopting dynamic, short-lived secrets on demand as a best practice easy. It leverages the same Pulumi Cloud identity, RBAC, Teams, SAML/SCIM, OIDC, and scoped access tokens used for Pulumi IaC to ensure secrets management complies with enterprise security policies. Every time secrets or configuration values are accessed or changed with Pulumi ESC, the action is fully logged for auditing. So you can trust (and prove) your secrets are secure. Pulumi ESC makes it easy to eliminate the need for developers to copy and paste secrets and store them in plaintext on their computers. Developers can easily access secrets via CLI, API, Kubernetes operator, the Pulumi Cloud UI, and in-code with Typescript/Javascript, Python, and Go SDKs.
```

Comment:
"Pulumi ESC makes adopting dynamic, short-lived secrets on demand as a best practice easy. " - typo? doesn't read right

Reply:
" It leverages the same Pulumi Cloud identity, RBAC, Teams, SAML/SCIM, OIDC, and scoped access tokens used for Pulumi IaC to ensure secrets management complies with enterprise security policies."

I'd love to just keep talk about the capabilities of RBAC, Team without saying "same"